src/katsdpcontroller/master_controller.py

```diff
@@ -1967,7 +1967,7 @@ def parse_args(argv: List[str]) -> argparse.Namespace:
     usage = "%(prog)s [options] zk singularity"
     parser = argparse.ArgumentParser(usage=usage)
     parser.add_argument(
-        "-a", "--host", default="", metavar="HOST", help="attach to server HOST [localhost]"
+        "-a", "--host", default="", metavar="HOST", help="bind to server HOST [any]"
```

Comment:
This comment was wrong from the start - not related to the functionality change in this PR.